src/Components/test/testassets/TestServer/Pages/Authentication.cshtml

```diff
@@ -0,0 +1,72 @@
+@page
+@using Microsoft.AspNetCore.Authentication
+@using System.Security.Claims
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Authentication</title>
+</head>
+<body>
+    <h1 id="authentication">Authentication</h1>
+    <p>
+        This is a completely fake login mechanism for automated test purposes.
+        It accepts any username, with no password.
+    </p>
+    <p>
+        <strong>Obviously you should not do this in real applications.</strong>
+        See also: <a href="https://docs.microsoft.com/aspnet/core/security">documentation on configuring a real login system.</a>
```

Comment:
Tbh I don't like this "Do not do this in a real application..." It would be good to build out a test example which shows are a near real world example.

Reply:
This is an automated test, not a sample.